suite/runsuite.cmake

```diff
@@ -318,7 +318,7 @@ string(REGEX MATCH "[^\n] \n" match "${diff_contents}")
 if (NOT "${match}" STREQUAL "")
   # Get more context
   string(REGEX MATCH "\n[^\n]+ \n" match "${diff_contents}")
-#NOCOMMIT  message(FATAL_ERROR "ERROR: diff contains trailing spaces: ${match}")
```

Comment:
Isn't this one of the keywords that actually prevents submission to master?

Reply:
Today it only looks for NOCHECKIN IIRC.  But it's tricky here b/c this has to be a word that does not hit any checks, in order to run the tests.  Maybe we should try to create some other way to skip certain up-front checks, if there were some way without touching the code...not sure that exists.